release/serve_tests/workloads/microbenchmarks.py

```diff
@@ -172,6 +163,12 @@ async def _main(
         if run_latency:
             channel = grpc.insecure_channel("localhost:9000")
             stub = serve_pb2_grpc.RayServeBenchmarkServiceStub(channel)
+            serve_grpc_options = gRPCOptions(
```

Comment:
This should probably be defined outside `if run_latency:` just under `if run_grpc:` else if we run this script manually with just grpc throughput, it will fail

Reply:
fixed!